index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Ingrse su nombre: ");
+alert(`Hola ${nombreUsuario}`);
```

Comment:
En este caso, me parece que seria mejor que no uses interpolacion de strings.
```suggestion
alert("Hola " + nombreUsuario);
```